data/ai/lua/ai_helper.lua

```diff
@@ -61,7 +61,7 @@ end
 
 function ai_helper.done_eval_messages(start_time, ca_name)
     ca_name = ca_name or 'unknown'
-    local dt = wesnoth.get_time_stamp() / 1000. - start_time
+    local dt = wesnoth.ms_since_init() / 1000. - start_time
```

Comment:
This local variable appears to be unused.

Reply:
I don't think that has anything to do with this PR though…

Reply:
Yeah, that's a leftover from a previous version of that function.  Feel free to remove it, or I can do so separately.  In the latter case, I'll wait until after this has been merged though, so that it doesn't cause conflicts.